src/app/proposals/hooks/useFetchLatestProposals.ts

```diff
@@ -60,12 +59,8 @@ const toFunctionSelector = ({ functionName, abi }: FunctionSelectorArgs) => {
 }
 
 const RELAY_FUNCTION_SELECTOR = toFunctionSelector({ abi: GovernorAbi, functionName: 'relay' })
-const CR_WHITELIST_FUNCTION_SELECTOR_MVP = toFunctionSelector({
```

Comment:
I don't think that we should remove this, since we also need to include the proposals from v1

Reply:
do we? I thought the whole point was to remove that